src/Console/Command/FixCommand.php

```diff
@@ -183,7 +183,7 @@ public function getHelp(): string
 * 64 - Exception raised within the application.
 
 EOF
-        ;
+            ;
```

Comment:
This change looks incorrect as the previous indentation was the expected one. Maybe we can remove the newline before `;` though.

Reply:
Of course, I'm reverting this. Btw., PHPStorm was very unfriendly with restoring this line.